build.zig

```diff
@@ -302,11 +302,7 @@ const exercises = [_]Exercise{
         .main_file = "057_unions3.zig",
         .output = "Insect report! Ant alive is: true. Bee visited 17 flowers.",
     },
-    .{
-        .main_file = "058_quiz7.zig",
-        .output = "Archer's Point--2->Bridge--1->Dogwood Grove--3->Cottage--2->East Pond--1->Fox Pond",
-        .hint = "This is the biggest program we've seen yet. But you can do it!"
-    },
+    .{ .main_file = "058_quiz7.zig", .output = "Archer's Point--2->Bridge--1->Dogwood Grove--3->Cottage--2->East Pond--1->Fox Pond", .hint = "This is the biggest program we've seen yet. But you can do it!" },
```

Comment:
This was changed from the formatter

Reply:
Ah, yes. I believe if you add the trailing comma to each of these lines items, `zig fmt` will understand that we wish them to be on separate items.  Could you do that with these and update the PR, please?

Thanks, I agree that this will look nicer. :+1: 

(My original plan was to have the output always be one short line per exercise, but that proved to be a difficult restriction to stick to.)